doc/manual/manual.tex

```diff
@@ -10316,9 +10302,9 @@ \subsubsection{Material models}
 starting point for implementations of new material models.
 
 Older versions of \aspect{} used to have individual functions like viscosity()
-instead of the evaluate() function discussed above. They are now a deprecated
-way of implementing a material model. You can get your old model working
-by deriving from InterfaceCompatibility instead of Interface.
+instead of the evaluate() function discussed above. This old interface is not longer
+supported, and you need to restructure your plugin to implement evaluate()
```

Comment:
just drop ", and you need to ...". This sentence is only true for a short period of time (because everyone will have updated it quickly) and it's not worth keeping in perpetuity.